cdap-ui/app/features/workflows/service/graph-helpers.js

```diff
@@ -118,9 +119,33 @@ angular.module(PKG.name + '.services')
       }
     }
 
+    function addStartAndEnd(nodes) {
```

Comment:
function still exists in es6. We just have an extra syntax (fat arrow functions) that allows us to express anonymous functions.